logfire/_internal/constants.py

```diff
@@ -145,6 +145,15 @@ def log_level_attributes(level: LevelName | int) -> dict[str, otel_types.Attribu
 RESOURCE_ATTRIBUTES_PACKAGE_VERSIONS = 'logfire.package_versions'
 """Versions of installed packages, serialized as list of json objects with keys 'name' and 'version'."""
 
+RESOURCE_ATTRIBUTES_CODE_GIT_REF = 'logfire.code.git_ref'
```

Comment:
In general we should try following semantic conventions where they exist. With a bit of effort I found https://opentelemetry.io/docs/specs/semconv/attributes-registry/vcs/. This would be `vcs.repository.ref.revision`.

Just noting that, not saying we should go with this overall approach or spend time on this now.

Reply:
Ah, I didn't find it. Nice. Thanks.

Reply:
They are experimental. They are not included in the `opentelemetry.semconv` package yet. What should I do?